src/libs/TransactionUtils.ts

```diff
@@ -630,6 +630,21 @@ function getEnabledTaxRateCount(options: TaxRates) {
     return Object.values(options).filter((option: TaxRate) => !option.isDisabled).length;
 }
 
+/**
+ * Check if the customUnitRateID has a value default for P2P distance requests
+ */
+
+function isCustomUnitRateIDForP2P(transaction: OnyxEntry<Transaction>): boolean {
+    return transaction?.comment?.customUnit?.customUnitRateID === CONST.CUSTOM_UNITS.FAKE_P2P_ID;
+}
+
+/**
+ * Get rate ID from the transaction object
```

Comment:
@neil-marcellini now better, or remove it completely? 😅

Reply:
Looks good 👍